features/step_definitions/user.rb

```diff
@@ -70,5 +82,12 @@
 end
 
 Then(/^(.*) should not be able to sign up$/) do |username|
-  expect(user_named(username).persisted?).to be_falsey
+  found_users = User.all.select do |user|
+    user.first_name.eql?(user_named(username).first_name) && user.last_name.eql?(user_named(username).last_name) && user.email.eql?(user_named(username).email)
+  end
+  expect(found_users.any?).to be_falsey
 end
+
+And(/^the page should show (.*)$/) do |message|
```

Comment:
This might cause some ambiguity later down the line. Could you wrap the parameter in quotes?